modules/collection/programs/ncmpcpp.nix

```
@@ -0,0 +1,93 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  inherit (lib.modules) mkIf;
+  inherit (lib.options) mkOption mkEnableOption mkPackageOption literalExpression;
+  inherit (lib.types) attrsOf listOf nullOr oneOf int str bool submodule;
+  inherit (lib.rum.generators.ncmpcpp) toNcmpcppBinding toNcmpcppSettings;
+  inherit (lib.rum.types.ncmpcpp) ncmpcppBindingType;
+
+  cfg = config.rum.programs.ncmpcpp;
+in {
+  options.rum.programs.ncmpcpp = {
+    enable = mkEnableOption "ncmpcpp";
+
+    package = mkPackageOption pkgs "ncmpcpp" {};
```

We could add extraDescription to mention that there are some settings that can be set as an overlay.
```suggestion
    package = mkPackageOption pkgs "ncmpcpp" {
      extraDescription = ''
        You can use an override to toggle certain features like the visualizer, a clock screen, and more.
        Please check out the package source for a complete list.
      '';
      example = ''(pkgs.ncmpcpp.override {
        visualizerSupport = true;
      })'';
    };
```

Something like this.

---

I didn't know about extraDescription lol, thx :3 I copied your text, if you don't mind.